Mailbox/Program.cs

```diff
@@ -89,17 +89,43 @@ static void Main(string[] args)
 
         public static string GetOwnersDisplay(Mailboxes mailboxes)
         {
-            
+            string ownersDisplay = "";
+
+            foreach (Mailbox mailbox in mailboxes) 
+            {
+                ownersDisplay += $"{mailbox.Owner.ToString()}\n";
```

Comment:
Inside of an interpolated string, it will convert the parameters to strings for you.
```suggestion
                ownersDisplay += $"{mailbox.Owner}\n";
```